libminifi/include/FlowControlProtocol.h

```diff
@@ -45,297 +51,269 @@ class FlowController;
-	RESP_SUCCESS,
-	RESP_TRIGGER_REGISTER, // Server respond to client report to re trigger register
-	RESP_START_FLOW_CONTROLLER, // Server respond to client to start flow controller
-	RESP_STOP_FLOW_CONTROLLER, // Server respond to client to stop flow controller
-	RESP_FAILURE,
-	MAX_RESP_CODE
+  RESP_SUCCESS,
+  RESP_TRIGGER_REGISTER,  // Server respond to client report to re trigger register
+  RESP_START_FLOW_CONTROLLER,  // Server respond to client to start flow controller
+  RESP_STOP_FLOW_CONTROLLER,  // Server respond to client to stop flow controller
+  RESP_FAILURE,
+  MAX_RESP_CODE
 } FlowControlRespCode;
 
 //! FlowControl Resp Code str
-static const char *FlowControlRespCodeStr[MAX_RESP_CODE] =
-{
-		"RESP_SUCCESS",
-		"RESP_TRIGGER_REGISTER",
-		"RESP_START_FLOW_CONTROLLER",
-		"RESP_STOP_FLOW_CONTROLLER",
-		"RESP_FAILURE"
-};
+static const char *FlowControlRespCodeStr[MAX_RESP_CODE] = { "RESP_SUCCESS",
+    "RESP_TRIGGER_REGISTER", "RESP_START_FLOW_CONTROLLER",
+    "RESP_STOP_FLOW_CONTROLLER", "RESP_FAILURE" };
 
 //! Flow Control Resp Code to String
-inline const char *FlowControlRespCodeToStr(FlowControlRespCode code)
-{
-	if (code < MAX_RESP_CODE)
-		return FlowControlRespCodeStr[code];
-	else
-		return NULL;
+inline const char *FlowControlRespCodeToStr(FlowControlRespCode code) {
+  if (code < MAX_RESP_CODE)
+    return FlowControlRespCodeStr[code];
+  else
+    return NULL;
 }
 
 //! Common FlowControlProtocol Header
 typedef struct {
-	uint32_t msgType; //! Msg Type
-	uint32_t seqNumber; //! Seq Number to match Req with Resp
-	uint32_t status; //! Resp Code, see FlowControlRespCode
-	uint32_t payloadLen; //! Msg Payload length
+  uint32_t msgType;  //! Msg Type
+  uint32_t seqNumber;  //! Seq Number to match Req with Resp
+  uint32_t status;  //! Resp Code, see FlowControlRespCode
+  uint32_t payloadLen;  //! Msg Payload length
 } FlowControlProtocolHeader;
 
 //! FlowControlProtocol Class
-class FlowControlProtocol
-{
-public:
-	//! Constructor
-	/*!
-	 * Create a new control protocol
-	 */
-	FlowControlProtocol(FlowController *controller) {
-		_controller = controller;
-		logger_ = Logger::getLogger();
-		configure_ = Configure::getConfigure();
-		_socket = 0;
-		_serverName = "localhost";
-		_serverPort = DEFAULT_NIFI_SERVER_PORT;
-		_registered = false;
-		_seqNumber = 0;
-		_reportBlob = NULL;
-		_reportBlobLen = 0;
-		_reportInterval = DEFAULT_REPORT_INTERVAL;
-		_running = false;
+class FlowControlProtocol {
+ public:
+  //! Constructor
+  /*!
+   * Create a new control protocol
+   */
+  FlowControlProtocol(FlowController *controller) {
+    _controller = controller;
+    logger_ = logging::Logger::getLogger();
+    configure_ = Configure::getConfigure();
+    _socket = 0;
+    _serverName = "localhost";
+    _serverPort = DEFAULT_NIFI_SERVER_PORT;
+    _registered = false;
+    _seqNumber = 0;
+    _reportBlob = NULL;
+    _reportBlobLen = 0;
+    _reportInterval = DEFAULT_REPORT_INTERVAL;
+    _running = false;
```

Comment:
Can you move "_" to end of class data member to adhere to Google C++ Style guide? Same for all variables here.